l10n_th_address/__manifest__.py

```diff
@@ -32,6 +32,5 @@
     ],
     'installable': True,
     'auto_install': False,
+    'post_init_hook': '_update_data_noupdate',
```

Comment:
Travis [did warn about that](https://travis-ci.org/OCA/l10n-thailand/jobs/228809869#L509) but suceeded nevertheless. That is not normal.